package.json

```diff
@@ -85,7 +86,7 @@
     "lint:fix": "gulp eslint --fix",
     "coverage": "gulp coveralls",
     "generate-docs": "node ./build/commands-yml/parse.js",
-    "shrinkwrap-prod": "rimraf package-lock.json && npm prune --production && npm shrinkwrap && npm install --no-shrinkwrap",
+    "shrinkwrap-prod": "rimraf package-lock.json && rimraf node_modules && npm install --production && npm shrinkwrap && node ./check-npm-pack-files && npm install --no-shrinkwrap",
```

Comment:
I'd personally prefer `shrinkwrap:prod`.

Reply:
Yeah, I like that syntax too. Change was pushed.